ios/FluentUI/Extensions/CharacterSet+Extension.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+
+extension CharacterSet {
+    static var whitespaceNewlineAndZeroWidthSpace: CharacterSet {
```

Comment:
What is this? I can't find it being used in this PR, and I can't seem to find it in vnext-tokens.

Reply:
Looks like some dead code from vnext-tokens branch that isn't even in our Xcode project. I'll remove it from here for now.